crawler/__init__.py

```diff
@@ -0,0 +1,30 @@
+from http import HTTPStatus
+import logging
+import logging.config
+import os
+
+from flask import Flask
+from flask_apscheduler import APScheduler
+
+scheduler = APScheduler()
+
+
+def create_app() -> Flask:
+    app = Flask(__name__)
+    app.config.from_object(os.environ["SETTINGS_MODULE"])
+
+    # setup logging
+    logging.config.dictConfig(app.config["LOGGING"])
+
+    if app.config.get("SCHEDULER_RUN", False):
+        scheduler.init_app(app)
+        scheduler.start()
+
+    @app.get("/health")
+    def health_check():
+        if scheduler.get_job("run_crawler"):
```

Comment:
The `run_crawler` job ID should probably be extracted as a constant so that if a change in made in the config it will be picked up here too.